src/main/java/com/google/jenkins/plugins/computeengine/client/ComputeClient.java

```diff
@@ -340,7 +339,7 @@ public int compare(Subnetwork o1, Subnetwork o2) {
 
     public Operation insertInstance(String projectId, String template, Instance instance) throws IOException {
         final Compute.Instances.Insert insert = compute.instances().insert(projectId, instance.getZone(), instance);
-        if (StringUtils.isNotEmpty(template)) {
+        if (!Strings.isNullOrEmpty(template)) {
```

Comment:
Both versions was checking exactly same things....

Reply:
Agreed, but we just have certain libraries we prefer.